Sources/WalletConnectAuth/Engine/Common/SessionEngine.swift

```diff
@@ -108,15 +108,18 @@ final class SessionEngine {
         }
     }
     
-    func request(params: Request) async throws {
-        print("will request on session topic: \(params.topic)")
-        guard sessionStore.hasSession(forTopic: params.topic) else {
-            logger.debug("Could not find session for topic \(params.topic)")
-            return
+    func request(_ request: Request) async throws {
+        print("will request on session topic: \(request.topic)")
```

Comment:
What the difference between use cases of `print` and `logger`?

Reply:
`logger` is meant to give a better control over logging categories when running the client, and replace `print`. But it still needs some improvements, overall.